integreat_cms/api/v3/locations.py

```diff
@@ -124,6 +124,9 @@ def locations(request, region_slug, language_slug):
         pois = pois.filter(location_on_map=location_on_map)
     for poi in pois:
         translation = poi.get_public_translation(language_slug)
+        poi.default_translation = poi.get_public_translation(
+            region.default_language.slug
+        )
```

Comment:
This changes the behavior of the response (because it now returns the title of the default public translation instead of the newest default translation.
Also, I find it a bit hacky to overwrite a read-only property - I would prefer it if we'd
1. Also prefetch the default translations by adding e.g. `.prefetch_translations()` to the queryset in line 118
2. (preferred) Add a new property `default_public_translation` to the abstract content model which works exactly like `default_translation` with the difference that it caches the default public translation instead of the newest default translation (which doesn't make sense for the API anyway, since we only want to deliver public translations). And then we use `default_public_translation` instead of `default_translation` in the API response.